docker.Makefile

```diff
@@ -110,7 +110,13 @@ lint: ## run linter(s)
 
 vendor: build_dev_image
 	$(info Update Vendoring...)
-	docker run --rm -v "$(CURDIR)":/go/src/github.com/docker/app/ $(DEV_IMAGE_NAME) make vendor
+	rm -rf ./vendor
+	docker rm -f docker-app-vendoring || echo ""
```

Comment:
Does `docker rm -f` exit with error if the container doesn't exist? That's a shame, since it means other more serious errors are also skipped.

The normal idiom here is `|| true` or `|| :`, but I expect those are portable enough?

Reply:
Yeah, || true should work I guess